src/components/DApps.js

```diff
@@ -25,10 +26,28 @@ class DApps extends React.Component {
 
   render() {
     const elements = this.state.data || [];
+    const newDApp = elements.filter((item) => {
+      if (item.category.name.toLowerCase() === 'new dapps') { return item; }
+      return null;
+    });
+    const othersDApp = elements.filter((item) => {
```

Comment:
can you rename? `othersDApp`?

`New DApps` always will have specific category ID, use that instead